README.md

```diff
@@ -334,16 +334,16 @@ petitionsclerk@example.com
 petitionsclerk1@example.com
 reportersOffice@example.com
 trialclerk@example.com
-judge.armen@example.com
-colvinsChambers@example.com
 judge.ashford@example.com
 ashfordsChambers@example.com
 judge.buch@example.com
 buchsChambers@example.com
-judge.carluzzo@example.com
+stjudge.carluzzo@example.com
```

Comment:
is this "st" intentional?

Reply:
Yes, it's for `special trial judge`. This is the email we use when creating the judge users in `judge_users.csv`, so we just updated this one to match.

Reply:
hm, but these users have already been created in the system and are being used. I guess I need to figure out what action needs to be done Court side for this, if any?

Reply:
Thanks @rachaelparris! Yeah, @julialeague , the one we have loaded in the system (and are using) is "stjudge.carluzzo@example.com". 👍